pkg/ipam/option/option.go

```diff
@@ -28,6 +28,10 @@ const (
 
 	// IPAMAlibabaCloud is the value to select the AlibabaCloud ENI IPAM plugin for option.IPAM
 	IPAMAlibabaCloud = "alibabacloud"
+
+	// IPAMDelegatedPlugin is the value to select CNI delegated IPAM plugin mode.
+	// See https://www.cni.dev/docs/spec/#section-4-plugin-delegation
+	IPAMDelegatedPlugin = "delegated-plugin"
```

Comment:
good call, will take a look at the helm chart as well

Reply:
I decided not to tackle helm chart / installer changes in this PR, since it's already pretty large. For testing, we've been using the existing helm chart options to install a CNI configuration from a configmap.

Reply:
Makes sense. It would be good to get that documented at some point so others can reproduce and test this setup.